packages/server/tests/utils/TestUtils.ts

```diff
@@ -354,12 +355,13 @@ export function initTestUtils(): TestUtilsReturnedValue {
         if (mpeRoomIDToAssociate !== undefined) {
             await Promise.all(
                 mpeRoomIDToAssociate.map(
-                    async ({ roomID, roomName: mpeRoomName }) => {
+                    async ({ roomID, roomName: mpeRoomName, isOpen }) => {
                         let mpeRoomToAssociate = await MpeRoom.find(roomID);
                         if (mpeRoomToAssociate === null) {
                             console.log('CREATING MPE ROOM FROM TEST UTILS');
                             mpeRoomToAssociate = await MpeRoom.create({
                                 uuid: roomID,
+                                isOpen: isOpen !== undefined ? isOpen : true,
```

Comment:
This is just a coding style concern and does not need to be changed, but this can also be written like this, using [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):

```suggestion
                                isOpen: isOpen ?? true,
```